### ISSUE TITLE
Sending digits in-call

### ISSUE THREAD

**Issue by GoogleCodeExporter** [BinaryBlue/imsdroid#66]
```
What steps will reproduce the problem?
1.
2.
3.

What is the expected output? What do you see instead?
In-call keypad for IVR navigation

What version of the product are you using? On what operating system?
Samsung Galaxy S

Please provide any additional information below.
Once in call there appears to be no way to send digits to navigate an IVR or 
voicemail system.

```

Original issue reported on code.google.com by `ant.mart...@gmail.com` on 23 Sep 2010 at 7:23

**Comment by GoogleCodeExporter** [comment]
```
Click on the "dialer" white box (bottom right) from the "In Call Screen".
In call Screen: 
http://code.google.com/p/imsdroid/wiki/Quick_Start#In_Call_Screen
```

Original comment by `boss...@yahoo.fr` on 23 Sep 2010 at 12:45
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the quick response.
Do you know if it is sending DTMF using inband tones, RFC2833 or SIP INFO
```

Original comment by `ant.mart...@gmail.com` on 23 Sep 2010 at 1:15

**Comment by GoogleCodeExporter** [comment]
```
It uses RFC4733 (Obsoletes RFC2833).
```

Original comment by `boss...@yahoo.fr` on 23 Sep 2010 at 1:39

**Comment by GoogleCodeExporter** [comment]
```
The developers of the platform I am using took a look a capture of the digits 
hitting the system. They observed that the DTMF’s are being seen on a 
different SSRC (0x005234a8) than the normal audio (0x7AEA1209) and that their 
system then ignored them.
```

Original comment by `ant.mart...@gmail.com` on 24 Sep 2010 at 9:32

**Comment by GoogleCodeExporter** [comment]
```
I managed to get a fix on the media server platform I am using to get around 
this issue.
```

Original comment by `ant.mart...@gmail.com` on 3 Nov 2010 at 3:46

**Comment by GoogleCodeExporter** [comment]
```
Fixed by "imsdroid-2.0.419-preview".
```

Original comment by `boss...@yahoo.fr` on 5 Jun 2011 at 5:19
- Changed state: **Fixed**
